platform/core/src/services/CustomizationService/CustomizationService.ts

```diff
@@ -234,19 +234,16 @@ export default class CustomizationService extends PubSubService {
   }
 
   /**
-   * A single reference is either an an array, or a single customization value,
-   * whose id is the id in the object, or the parent id.
-   * This allows for general use to register customizationModule entries.
+   * A single reference is either an string to be loaded from a module,
```

Comment:
The fixes in this file were required in order to test segmentation on the multi-volume view - they were causing infinite recursion due to an incorrect assumption about how the items are defined.